### ISSUE TITLE
create() doesn't return created object

### ISSUE THREAD

**Issue by Toilal** [demonsters/prisma-mock#29]
I have this simple code, but `created` variable seems to be defined with first existing record instead of newly created record.

```typescript
const created = await prisma.jobTitle.create({
  data: {
    label: 'dummy',
  },
});
```

**Comment by Toilal** [comment]
I try to reproduce in your unit tests, and it seems to work properly. Maybe it's because i use uuid as @id inside this entity model.

**Comment by stijnvanderlaan** [comment]
Fixed in 0.6.0
